File: postorder traversal.py
class BinaryTree:
  def __init__(self, data):
    self.data = data
    self.left = None
    self.right=None
    
def insert(root,Value):
    
    if root is None:
        root=BinaryTree(Value)
        return root

    if Value<root.data:
        root.left=insert(root.left,Value)
    else:
        
        root.right=insert(root.right,Value)
    return root
def postorder(root):
    
        if root==None:
            return
        
        postorder(root.left)
        
        postorder(root.right)  
        
        print(root.data)                 
root= insert(None,20)
insert(root,66)
insert(root,33)
insert(root,56)
insert(root,11)
insert(root,18)
insert(root,90)
print("Printing values of binary tree in postorder Traversal.")
postorder(root)

File: postorder traversal.cpp
#include <bits/stdc++.h>
using namespace std;
class T {
    public:
        int data;
        T* left = NULL,*right = NULL;
        
        T(int y) {
            data = y;
            left = NULL;
            right = NULL;
        }
};
void postorder_traversal (T *root) {
    if (root == NULL) return;
    
    postorder_traversal(root -> left);
    
    postorder_traversal(root -> right);
   
    cout << root -> data << ” “;
}
int main() {
    T *root = new T(14);
    root -> left = new T(22);
    root -> right = new T(66);
    root -> left -> left = new T(6);
    root -> left -> right = new T(84);
    root -> left -> left -> left = new T(2);
    root -> right -> left = new T(19);
    postorder_traversal(root);
    return 0;
}
